nifi-extension-bundles/nifi-iceberg-bundle/nifi-iceberg-common/pom.xml

```diff
@@ -86,6 +89,21 @@
             <artifactId>iceberg-orc</artifactId>
             <version>${iceberg.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-aws</artifactId>
+            <version>${iceberg.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-azure</artifactId>
+            <version>${iceberg.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-gcp</artifactId>
+            <version>${iceberg.version}</version>
+        </dependency>
```

Comment:
@mark-bathori Are these additional dependencies required as part of this change? They appear to be unrelated, and introduce a potentially large set of additional transitive dependencies.

Reply:
These dependencies contains different cloud provider IO implementations: S3FileIO, GCSFileIO and ADLSFileIO